modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -398,134 +375,134 @@ Copies a source object to a new destination; to copy an object, the caller's acc
 ** AUTHENTICATED_READ
 ** LOG_DELIVERY_WRITE
 ** BUCKET_OWNER_READ
-** BUCKET_OWNER_FULL_CONTROL |  The acl of the destination object. |  PRIVATE |
+** BUCKET_OWNER_FULL_CONTROL |  +++The acl of the destination object.+++ |  +++PRIVATE+++ | 
 | Destination Storage Class a| Enumeration, one of:
 
 ** STANDARD
 ** STANDARD_IA
 ** INTELLIGENT_TIERING
 ** ONEZONE_IA
 ** REDUCED_REDUNDANCY
-** GLACIER |  one of StorageClass enumerated values, defaults to StorageClass#STANDARD |  STANDARD |
-| Destination User Metadata a| Object |  The new metadata of the destination object, that if specified, overrides that copied from the source object |  |
-| Modified Since a| DateTime |  The modified constraint that restricts this request to executing only if the object has been modified after the specified date. This constraint is specified but does not match, no copy is performed |  |
-| Unmodified Since a| DateTime |  The unmodified constraint that restricts this request to executing only if the object has not been modified after this date. This constraint is specified but does not match, no copy is performed |  |
-| Encryption a| String |  Encryption method for server-side encryption. Supported value AES256. |  |
-| Content Type a| String |  The destination object's Content-Type HTTP header |  |
-| Target Variable a| String |  The name of a variable that stores the output of an operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the targer variable. |  `#[payload]` |
+** GLACIER |  +++one of StorageClass enumerated values, defaults to StorageClass#STANDARD+++ |  +++STANDARD+++ | 
```

Comment:
Is storageClass#STANDARD correct in the published doc (View File view)?

Reply:
It should show just `STANDARD`, probably some bug in markdown generation from javadocs. However, we should not modify it in code as it's correct in Javadoc. We can modify it by hand here but we will need to do it after every release.

Reply:
Please modify this by hand.

Reply:
Fixed.